Ex_21/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+alert ('Hello! How are you?')
+let firstNum = prompt ('Enter first number, please', 100);
+console.log( 'First number is: ', firstNum);
+
+let secondNum = prompt ('Enter second number, please', 100);
+console.log( 'Second number is: ', secondNum);
+let result = firstNum > secondNum;
```

Comment:
When you are comparing numbers, as a best practice get them converted to numbers